src/MicrosoftTeams.interface.ts

```diff
@@ -0,0 +1,13 @@
+export const enum NotificationTypes {
+  fileDownloadStart = "fileDownloadStart",
+  fileDownloadComplete = "fileDownloadComplete"
+}
+
+export interface ShowNotificationParameters {
+  message: string;
+  notificationType: NotificationTypes;
+}
+
+export interface ExecuteDeepLinkParameters {
+  deepLink: string;
+}
```

Comment:
Why only move these as opposed to all the enums/interfaces from MicrosoftTeams.ts?

Reply:
Will move other in separate PR